libfwupdplugin/fu-hwids.c

```diff
@@ -379,7 +379,7 @@ fu_hwids_setup_overrides(FuHwids *self, GError **error)
 	for (guint i = 0; i < keys->len; i++) {
 		const gchar *key = g_ptr_array_index(keys, i);
 		g_autofree gchar *value = g_key_file_get_string(kf, "fwupd", key, NULL);
-		if (value != NULL)
+		if (value != NULL && strlen(value))
 			fu_hwids_add_smbios_override(self, key, value);
```

Comment:
how do we set keys as zero length strings? e.g. setting the Family to `""`?

Reply:
I thought about that; but is that something that we often need to do though?

Reply:
OK, I guess these can't be done that way then.  I'll revert this part of the series.